lib/vdsm/storage/blockVolume.py

```diff
@@ -366,7 +365,6 @@ def calculate_volume_alloc_size(
                                               "volume_utilization_chunk_mb")
                 alloc_size = chunk_size_mb * MiB
         else:
-            # Preallocated qcow2
```

Comment:
This comment is good, but it is missing the raw-preallocated case. Instead of removing it, we need to list the possible cases:

     Preallocated raw or qcow2